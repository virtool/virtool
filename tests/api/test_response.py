async def test_not_found(spawn_client, snapshot):
    """Test that a 404 is returned when the job doesn't exist."""
    client = await spawn_client(authenticated=True)

    resp = await client.put("/foobar", data={})

    assert resp.status == 404
    assert await resp.json() == snapshot
