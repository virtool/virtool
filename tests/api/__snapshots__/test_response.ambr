# serializer version: 1
# name: test_not_found
  dict({
    'id': 'not_found',
    'message': 'Not found',
  })
# ---
